README.md

```diff
@@ -78,6 +78,64 @@ Since imgix can generate as many derivative resolutions as needed, ember-cli-img
 
 NB: Since this library sets [`fit`](https://docs.imgix.com/apis/url/size/fit) to `crop` by default, when just a width or height is set, the image will resize and maintain aspect ratio. When both are set, the image will be cropped to that size, maintaining pixel aspect ratio (i.e. edges are clipped in order to not stretch the photo). If this isn't desired, set `fit` to be another value (e.g. `clip`)
 
+#### Lazy Loading
+
+If you'd like to lazy load images, we recommend using [lazysizes](https://github.com/aFarkas/lazysizes). In order to use ember-cli-imgix with lazysizes, you can simply tell it to generate lazysizes-compatible attributes instead of the standard `src`, `srcset`, and `sizes` by changing some configuration settings:
+
+```jsx
+// config/environment.js
+
+module.exports = function(environment) {
+  var ENV = {
+    // snip
+    APP: {
+      imgix: {
+        attributeNameMap: {
+          src: 'data-src',
+          srcset: 'data-srcset'
+        }
+      }
+    }
+    // snip
+  };
+};
+```
+
+Otherwise, you can use the component as normal.
+
+**NB:** It is recommended to use the [attribute change plugin](https://github.com/aFarkas/lazysizes/tree/gh-pages/plugins/attrchange) in order to capture changes in the data-\* attributes. Without this, changing the attributes to this library will have no effect on the rendered image.
```

Comment:
I feel like this should be worded a bit more strongly? "Recommended" sounds too soft for something that is required to make it work.

Reply:
It's not required - the developer is able to `lazySizes.init()` after every update if they would like as well.